dashboard/src/pages/ingest/upload/index.vue

```diff
@@ -0,0 +1,54 @@
+<script setup lang="ts">
+import Uppy from "@uppy/core";
+import { Dashboard } from "@uppy/vue";
+import XHR from "@uppy/xhr-upload";
+import { useRouter } from "vue-router/auto";
+
+import { getPath } from "@/client";
+import { useAuthStore } from "@/stores/auth";
+import { useLayoutStore } from "@/stores/layout";
+import IconUpload from "~icons/clarity/plus-circle-line";
+
+import "@uppy/core/dist/style.min.css";
+import "@uppy/dashboard/dist/style.css";
+import "@uppy/progress-bar/dist/style.min.css";
+
+const router = useRouter();
+const authStore = useAuthStore();
+const layoutStore = useLayoutStore();
+
+const GB = 1000 ^ 3;
+const maxFileSize = 500 * GB;
+
+layoutStore.updateBreadcrumb([{ text: "Ingest" }, { text: "Upload SIPs" }]);
+
+const uppy = new Uppy({ restrictions: { maxFileSize: maxFileSize } }).use(XHR, {
+  endpoint: getPath() + "/ingest/sips/upload",
+  headers: { Authorization: `Bearer ${authStore.getUserAccessToken}` },
+  allowedMetaFields: false,
+  getResponseData: () => {
+    return { url: "" };
+  },
+  onAfterResponse: (xhr) => {
+    if (xhr.status === 201) {
+      setTimeout(() => {
+        router.push({
+          path: "/ingest/sips",
+        });
+      }, 500);
+    }
+  },
```

Comment:
This will redirect after the first response is received, but why the timeout?

Reply:
I found the immediate redirect after the upload completes to be jarring, and I prefer the brief pause.  We can see how this tests, and adjust or eliminate the timeout if the feedback is negative.

I think once #1210 is implemented we can return a URL for the uploaded SIP and remove this redirect altogether.